tests/test_edc.py

```diff
@@ -7,10 +7,36 @@
 from numpy import array
 
 import numpy.testing as npt
+import pyfar as pf
+import pytest
 
 import pyrato as ra
 
 
+@pytest.mark.parametrize("is_energy", [True, False])
+def test_schroeder_integration(is_energy):
```

Comment:
Maybe add an explcit assertion of `energy_decay_curve.time.shape == data.time.shape`